README.md

```diff
@@ -56,6 +61,25 @@ Option | Description | Default
 `prosopite_ignore_queries` | Sets the `ignore_queries` option for prosopite. | `[/schema_migrations/i]`
 `content_security_policy_nonce` | Sets the content security policy nonce to use when inserting Dial's script. Can be a string, or a Proc which receives `env` and response `headers` as arguments and returns the nonce string. | Rails generated nonce or `nil`
 
+## Comparison with [rack-mini-profiler](https://github.com/MiniProfiler/rack-mini-profiler)
+
+|                           | rack-mini-profiler                 | Dial                                                    |
+| :------------------------ | :--------------------------------- | :------------------------------------------------------ |
+| Compatibility             | Any Rack application               | Only Rails applications                                 |
+| Database Profiling        | Yes                                | Yes (via vernier hook - marker table, chart)            |
+| N+1 Query Detection       | Yes* (needs to be inferred)        | Yes (via prosopite)                                     |
+| Ruby Profiling            | Yes (with stackprof - flame graph) | Yes (via vernier - flame graph, stack chart, call tree) |
+| Ruby Allocation Profiling | Yes (with stackprof)               | Yes (via vernier)                                       |
+| Memory Profiling          | Yes (with memory_profiler)         | Yes* (via vernier hook - overall usage only, graph)     |
+| View Profiling            | Yes                                | Yes (via vernier hook - marker table, chart)            |
+| Snapshot Sampling         | Yes                                | No                                                      |
+| Production Support        | Yes                                | No                                                      |
```

Comment:
is this a hard no? Seems like the overhead of vernier would be smaller than stackprof? I guess it's up to the end user to decide if they want to add to production and wear the sampler costs?

Reply:
> is this a hard no? 

No, it just means that it's not currently supported, namely the ability to persist and serve profiles across distributed machines.

> Seems like the overhead of vernier would be smaller than stackprof?

[I suspect so](https://github.com/jhawthorn/vernier/issues/65).

> I guess it's up to the end user to decide if they want to add to production and wear the sampler costs?

Yes. I first need to add an enable/disable config (it's currently enabled by default if `require`ed) as I suspect most users would like to disable by default in production and then enable via a param/write their own sampling logic. Will add these in future releases. For now, I want to explicitly advise against using it in production.